user/config/config_gotls.go

```diff
@@ -311,8 +305,18 @@ func (gc *GoTLSConfig) ReadTable() (*gosym.Table, error) {
 		return nil, fmt.Errorf("could not find magic number in %s ", gc.Path)
 	}
 	tableData = tableData[pclntabIndex:]
-
-	addr := gc.goElf.Section(".text").Addr
+	var addr uint64
+	{
+		// get textStart from pclntable
+		// please see https://go-review.googlesource.com/c/go/+/366695
+		// tableData
+		ptrSize := uint32(tableData[7])
```

Comment:
Is 7 a fixed index position?

Reply:
yes, please see [pclntab.go#L249](https://github.com/golang/go/blob/master/src/debug/gosym/pclntab.go#L249), ` ptrSize are the same between 1.2, 1.16, and 1.18`